src/components/CommonTodo/View.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import FunctionButtons from '../functionButtons';
 import { Draggable  } from "react-beautiful-dnd";
-import MyDueTime from '../MyDueTime'
+import MyDueTime from '../MyDueTime';
+import store from '../../redux/store'
 
 class View extends React.Component {
     constructor(props){
         super(props);
        
         this.convertDataToClassName = this.convertDataToClassName.bind(this);
-       // this.saveDate = this.saveDate.bind(this);
+        this.doubleClick = this.doubleClick.bind(this);
```

Comment:
maybe consider some other names, try to not use click, double click for function naming